README.md

```diff
@@ -132,9 +133,9 @@ Successful response:
 <br>
 
 ___
-- A number param can be provided to return more or less than the default 4 word choices
+- An integer param can be provided to return more or less than the default 4 word choices
```

Comment:
Nice job here pointing specifically to an _integer_ rather than just a _number_. Note:
- would it be good to specify a range as well? Ex. "A integers param between 1 - x...," x being the max?
   - should we cap "x"...? 
      - if yes, at what number? 
      - if no, why not? why would someone want, say, 100 words?

Reply:
Good call. 

- Right now, we only have 80 words in the database. So games with any more than 20 rounds would be duplicating words (and since it's random, odds are that even 10 rounds could see duplicate words.) 
- I can add more words to database, but maybe we should limit to 10 rounds for the time being.
- Thoughts?

Reply:
Hey, thanks for the thoughts.

Note:  my question was related more to setting **guardrails** in relation to the params we're accepting. Yes, the database is related to this as it is our source but I don't think we need to add words atm or let it distract us from the initial point.

Sorry for not being more clear. Initially I was referencing the word count params (_num_). 
- **max**: I was asking if this should be limited so that someone can't request, say, 100 words (regardless of how large our database is)
- **min**: should there be a minimum amount of words they can request (what happens if the params is "num=-1" or "num=0"?
   - for this reason I suggested that we have a _num_ range

quick thought which you brought up above:
- we're not tracking rounds (that I know of)
  - thus, how would we limit the number of rounds?
     - having "users" is not MVP, so we'll proceed for now and circle back to this later to track rounds - unless you know of how we can track who the request is coming from
       - maybe _sessions_ and a timer (ex keep sessions data for y number of hours?) idk... also dont know how easy and quick of a lift this would be